packages/ethereum-contracts/package.json

```diff
@@ -74,7 +74,7 @@
         "build:post-contracts": "run-p -l build:post-contracts:*",
         "build:post-contracts:abi-bundle": "tasks/build-bundled-abi.sh",
         "build:post-contracts:dev-scripts-typings": "rm -rf dev-scripts/*.d.ts dev-scripts/*.d.ts.map; tsc -p tsconfig.scripts.json",
-        "build:post-contracts:contracts-size": "forge build --sizes > build/contracts-sizes.txt",
+        "build:post-contracts:contracts-size": "forge build && forge build --sizes > build/contracts-sizes.txt || true",
```

Comment:
we should remove this workaround.

Reply:
I changed this task such that it ignores contracts used only in (ephemeral) test context.
That way it will still fail if any other contract gets "oversized".